homeassistant/scripts/ensure_config.py

```diff
@@ -30,7 +30,7 @@ def run(args):
     # Test if configuration directory exists
     if not os.path.isdir(config_dir):
```

Comment:
Can this go?

Reply:
We should still have that check in case the user maps in a file instead of a directory to the container